src/main/java/frc/robot/commands/PrintCommand.java

```diff
@@ -0,0 +1,28 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/latest/docs/software/commandbased/convenience-features.html
+public class PrintCommand extends InstantCommand {
+  private String message;
+  public PrintCommand(String message) {
+    this.message = message;
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    SmartDashboard.putString("Test Slot", message);
+  }
+}
```

Comment:
Interesting.  PrintCommand is also a library class.  If we want to keep this, lets rename ours.

Reply:
yeah we don't need to keep this, however it was an incredibly helpful tool to test both shuffleboard and button declaration